src/components/popup/details/vendors/vendors.less

```diff
@@ -73,20 +73,20 @@ div.vendors {
 	  	text-align: center;
 	  	min-width: 60px;
 
-	 	@media @smartphone {
-			min-width: 65px;
+	  	span:first-child {
```

Comment:
I think that real problem is that is not allowed to put `span` tag directly inside `tr`. Only `td` and `th` is allowed there. We should fix this issue instead doing hacks i CSS because such HTML is invalid so we never know when it will stop working in some future browsers version.